precheck/lib/precheck/rule_processor.rb

```diff
@@ -28,10 +28,12 @@ def should_trigger_user_error?
 
     def has_errors_or_warnings?
       return true if error_results.length > 0 || warning_results.length > 0
+      return false
     end
 
     def items_not_checked?
       return true if items_not_checked.length > 0
```

Comment:
I think it would need to be `return error_results.length > 0 || warning_results.length > 0` which didn't seem as readable to me.